api/src/wfl/service/slack.clj

```diff
@@ -12,6 +12,8 @@
 (defonce ^:private token
   (delay (:bot-user-token (#'env/vault-secrets "secret/dsde/gotc/dev/wfl/slack"))))
 
+(def feature-env-var-name "WFL_SLACK_FEATURE_SWITCH")
```

Comment:
This could name the sense of the switch: WFL_SLACK_ENABLED or WFL_SLACK_DISABLED
Do we want this in wfl.environment/defaults?

Reply:
> This could name the sense of the switch: WFL_SLACK_ENABLED or WFL_SLACK_DISABLED

I took this approach originally with `WFL_SLACK_ENABLED` but got a bit turned around with how `wfl.environment/getenv` deals with env variable overrides in tests.  Can talk more if you're interested.

>Do we want this in wfl.environment/defaults?

It is not strictly necessary, but I'll add it in.  No harm in being explicit.

Reply:
Now `WFL_SLACK_ENABLED`.